docs/examples/kwok-manage-nodes-and-pods.md

```diff
@@ -0,0 +1,107 @@
+# Manage nodes and pods with kwok
+
+## Kwok with args `--manage-all-nodes=true`
+
+Kwok will keep all nodes in the cluster ready
+
+## Kwok with args `--manage-nodes-with-annotation-selector=kwok.x-k8s.io/node=fake`
+
+Kwok will keep all nodes with annotaion `kwok.x-k8s.io/node=fake` in the cluster ready
+
+## Create a node
+
+Node's status can be any value you want
```

Comment:
With kowk, you can join arbitrary Node(s) just by simply creating `v1.Node` object(s):

> The `status` can be any value.